code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -538,9 +540,11 @@
 	if(user.body_position == LYING_DOWN && get_dist(src, user) > 0)
 		return
 
-	if(!toggle(user))
-		togglelock(user)
+	if(toggle(user))
+		return
```

Comment:
![image](https://user-images.githubusercontent.com/6209658/184504860-5ae67754-2bd4-459a-83a7-04a8b24d1f91.png)

Reply:
I added an extra Opened check in this part. I couldn't replicate this code being reached while the locker was opened though because the tool is just put in the locker instead before ever reaching this.